tests/test_utils.py

```diff
@@ -0,0 +1,101 @@
+from typing import List
+
+import pytest
+
+from hatch_openzim.utils import get_github_project_homepage, get_python_versions
+
+
+@pytest.fixture
+def mock_check_output(mocker):
+    return mocker.patch("subprocess.check_output", autospec=True)
+
+
+@pytest.mark.parametrize(
+    "path_to_repo, git_url, expected_homepage_url",
+    [
+        (
+            "/path/https/repo1",  # this must change between test cases due to lru_cache
+            b"https://github.com/oneuser/onerepo.git\n",
```

Comment:
Why is the git_url `bytes`? Aren't we returning `str` from the function?

Reply:
This is what was returned by `subprocess.check_output`, to be used by the mock.